File: com.getshop.messages/src/com/thundashop/core/usermanager/data/Group.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.usermanager.data;

import com.thundashop.core.common.DataCommon;
import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author ktonder
 */
public class Group extends DataCommon {
    public String groupName = "";
    public String imageId = "";
    
    public List<Address> extraAddresses = new ArrayList();
    public Address defaultDeliveryAddress = new Address();
    public Address invoiceAddress = new Address();
    
}

File: com.getshop.client/events/core/usermanager/data/Group.php
<?php
class core_usermanager_data_Group extends core_common_DataCommon  {
	/** @var String */
	public $groupName;

	/** @var String */
	public $imageId;

	/** @var String */
	public $extraAddresses;

	/** @var core_usermanager_data_Address */
	public $defaultDeliveryAddress;

	/** @var core_usermanager_data_Address */
	public $invoiceAddress;

}
?>